packages/jelos/sources/scripts/runemu.sh

```diff
@@ -454,15 +454,18 @@ fi
 clear_screen
 
 ### Restore the overclock mode
-OVERCLOCK=$(get_setting "system.overclock")
```

Comment:
This is another unrelated bug fix -- saw this come up a few times
```
# cat /var/log/es_launch_stderr.log
/usr/bin/runemu.sh: line 460: /usr/bin/overclock: No such file or directory
```

I think it's benign, since it happens on shutdown but still, best if we don't blow up

Reply:
This is important on the RG552, but this fix looks good.  ty :)